src/service-providers/mailchimp/index.js

```diff
@@ -67,25 +56,37 @@ const renderPreSendInfo = newsletterData => {
 			<br />
 			<strong>{ listData.name }</strong>
 			<br />
-			{ listData.groupName && (
-				<Fragment>
-					{ __( 'Group:', 'newspack-newsletters' ) } <strong>{ listData.groupName }</strong>
+			{ sublistData && (
+				<>
+					{ sublistData.entity_type.charAt(0).toUpperCase() + sublistData.entity_type.slice(1) + ': '}
+					<strong>{ sublistData.name }</strong>
 					<br />
-				</Fragment>
+				</>
 			) }
 			<strong>
 				{ sprintf(
 					// Translators: subscriber count help message.
-					_n( '%d subscriber', '%d subscribers', listData.subscribers, 'newspack-newsletters' ),
-					listData.subscribers
+					_n( '%d subscriber', '%d subscribers', subscriberCount, 'newspack-newsletters' ),
+					subscriberCount
```

Comment:
This is not considering the selected group/segment/tag and I'm getting the total subscribers from the audience on the pre-send modal:

<img width="255" alt="image" src="https://github.com/user-attachments/assets/f3d3080a-5e65-418a-bacb-bb18c5d5f5ed">

<img width="238" alt="image" src="https://github.com/user-attachments/assets/2f48636f-af1f-479d-ac1a-5501e240c2e5">

Reply:
Ah, good catch—this is because the pre-send modal for MC only was using the `campaign.recipients.recipient_count` value from `newsletterData`, but this isn't updated until after a sync is completed AND we rehydrate the Redux store with another `retrieve` request. To avoid the miscount and also not add another roundtrip request delay before showing the modal, 3ffcd6b refactors this to use the `Send_List` and `Send_Sublist` counts, just like in the sidebar summary. There's a small chance that this count could be slightly off if the count changes in the ESP after the most recent `retrieve`, but the difference would likely be negligible, and since clicking the "Send" button does trigger a post save + campaign sync + MJML re-render + `retrieve` request + store rehydration, any incorrect value would likely update itself after a few seconds.